src/main/scala/zio/kafka/admin/AdminClient.scala

```diff
@@ -445,6 +453,27 @@ object AdminClient extends Accessible[AdminClient] {
       }
     }.map(_.asScala.toMap.bimap(TopicPartition(_), ListOffsetsResultInfo(_)))
 
+    /**
+     * List offset for the specified partitions.
+     */
+    override def listOffsetsAsync(
+      topicPartitionOffsets: Map[TopicPartition, OffsetSpec],
+      options: Option[ListOffsetsOptions]
+    ): Task[Map[TopicPartition, Task[ListOffsetsResultInfo]]] = {
+      val asJava                = topicPartitionOffsets.bimap(_.asJava, _.asJava).asJava
+      val topicPartitionsAsJava = topicPartitionOffsets.keySet.map(_.asJava)
```

Comment:
🟢 to avoid one `asJava`

```suggestion
      val topicPartitionOffsetsAsJava = topicPartitionOffsets.bimap(_.asJava, _.asJava)
      val topicPartitionsAsJava       = topicPartitionOffsetsAsJava.keySet
      val asJava                      = topicPartitionOffsetsAsJava.asJava
```